### DISCUSSION TITLE
PatchTool 6.11 5/22

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#172] [Announcements]
修正 #166 #167 #168

檔案：
- FFXIVChnTextPatch-0522.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-0522.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- [backup-0508.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.62/backup-0508.rar)：乾淨的0a0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。
- [rawexd-ja-611-0508.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.62/rawexd-ja-611-0508.rar)：日文原文的CSV。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.63'>PatchTool 6.11 5/22</a>.</em>

**Comment by KppKerry** [comment]
辛苦大大了 非常感謝

**Comment by tonyjediex** [comment]
謝謝大哥 辛苦了~

**Comment by siler2011** [comment]
謝謝大大!

**Comment by stevenbla** [comment]
![image](https://user-images.githubusercontent.com/106292097/170406941-4536648f-cadf-4c4d-a914-d5c0b2e035c1.png)
![image](https://user-images.githubusercontent.com/106292097/170407151-a45d6c81-31fc-4728-ba60-70e354fbb013.png)
請問要怎麼解決
